action-bump-version/action.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bump patch version
+description: |
+  Bump the current patch version in file VERSION
+  and open a new PR with the changes.
+  file VERSION is expected to the patch number
+  on a single line
+  ...
+
+inputs:
+  current-version:
+    required: true
+    description: the current version to bump
+  patch-version-identifier:
+    default: VERSION_PATCH
+    description: |
+      the name of the variable that sets the patch version in the ./VERSION file
```

Comment:
the `./VERSION` file path should be customizable

Reply:
no, my point here is to enforce it, so all xcore components use the same format :smiling_imp: